docs/zh-CN/docs/sql-manual/sql-functions/aggregate-functions/group_uniq_array.md

```diff
@@ -0,0 +1,80 @@
+---
+{
+    "title": "GROUP_UNIQ_ARRAY",
+    "language": "zh-CN"
+}
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## GROUP_UNIQ_ARRAY
+### description
```

Comment:
GROUP_UNIQ_ARRAY seems is same as collect_set?
why not impl max_size on collect_set function

Reply:
May we talk about it on wechat, I added your wechat from @Yukang-Lian just now.